src/routes/login/+page.server.ts

```diff
@@ -17,7 +21,61 @@ export const load: PageServerLoad = async ({ locals, request }) => {
         redirect(302, "/setup-wizard");
     }
 
-    const config = await getConfig();
+    /* Header authentication */
+    if ((env.HEADER_AUTH_ENABLED ?? "false") == "true" && !!env.HEADER_USERNAME) {
+        const username = request.headers.get(env.HEADER_USERNAME);
+        if (username) {
+            let user = await client.user.findUnique({
+                select: {
+                    id: true
+                },
+                where: {
+                    username: username
+                }
+            });
+
+            if (!user) {
+                if (!env.HEADER_EMAIL || !env.HEADER_NAME) {
+                    return fail(400, { username: username, password: "", incorrect: true });
+                }
+                const name = request.headers.get(env.HEADER_NAME);
+                const email = request.headers.get(env.HEADER_EMAIL);
+                if (!name || !email) {
+                    return fail(400, { username: username, password: "", incorrect: true });
```

Comment:
Maybe add a log message here as well that the required headers are missing from the request

Reply:
Yes, I will fix!